vendor/flarum/tags/less/common/TagLabel.less

```diff
@@ -60,3 +59,37 @@
     }
   }
 }
+@media (max-width: 768px){
+  .TagLabel {
+    font-size: 85%;
+    font-weight: 545;
+    padding: 0.1em 0.2em;
+  }
+  .TagsLabel .TagLabel {
+    &:nth-child(3){
+        border-radius: 0 @border-radius @border-radius 0;
+        .TagLabel-text >i::before{
+        display: none;
+      }
+      .TagLabel-text >i::after{
+        content: '...';
```

Comment:
Chỗ này em có thể sử dụng `text-overflow: elipsis` và `white-space: no-wrap` để hiện dấu `...` và nó sẽ scale theo chiều dài của cái class. 

Em có thể đọc thêm về `elipsis` tại [đây](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow) và `white-space: no-wrap` tại [đây](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space).

Reply:
Dạ em có thử dùng overflow: hidden, text-overflow: elipsis thì nó có scale theo chiều dài class nhưng mà nó bị lỗi css 
![image](https://user-images.githubusercontent.com/56894648/104105175-03cec800-52df-11eb-9263-d8b99008edc7.png)
với chiều dài class cố định khi hiển thị qua các màn hình khác nhau, nên nó có thể bị icon phía sau đè lên, nên em thấy để như cũ nó hợp lý hơn ạ

Reply:
Anh đoán là em ứng dụng 3 cái css của anh cho TagsLabel có phải không? nếu đúng thì việc em có như trên là do span luôn luôn scale theo chữ bên trong nếu như được xài inline (em có thể đọc thêm ở [đây](https://stackoverflow.com/questions/257505/css-fixed-width-in-a-span)). Do đó nó lúc nào cũng bao nhiêu đó chiều dài và không thể overflow hay cắt được . Cái em thấy cắt đó là do cái parent tag của cái span TagsLabel.

Để giải thì em nên đem mấy cái css đó lên 1 cấp bậc nữa là áp dụng cho cái li tag có cái class là item-tags. Còn background và border thì trên điện thoại, em biết là chỉ có thể hiện cao lắm 2 cái tag dựa theo những gì em làm bên dưới, và đồng thời em có thể biết được có bao nhiêu phần tử ở trong tags tại [đây](https://github.com/huuduy216/Talkie-Web/blob/master/vendor/flarum/tags/js/src/forum/addTagLabels.js#L42-L44) . Như vậy thì em chỉ cần chèn thêm 1 cái class để em biết có bao nhiêu phần tử mà chỉnh css cho cái li tag đó.

Nếu em cần anh giải thích thêm thì cứ nhắn cho anh. Anh sẽ sẵn sàng giải thích thêm nếu em còn thắc mắc.